scroll-func.js

```diff
@@ -0,0 +1,10 @@
+let scrollBtn = document.querySelector(".back-to-top");
+window.addEventListener("scroll", function(){
+      let position = window.scrollY;
+      if(position < 200){
+      scrollBtn.style.opacity = "0";
```

Comment:
@cesswairimu should not we use "visibility=hidden" instead of "opacity=0" as in this case we still are able to interact with the "scrollBtn"

Reply:
@frappelatte28 I thought the two are the same...but I am not the best at styling :laughing: ...would you like to give it a shot and open a PR with the visibility option, see how it goes?